demos/src/js/chartAuthoring-demo.js

```diff
@@ -0,0 +1,59 @@
+var demo = demo || {};
+
+/*
+
+Potential demo color stacks:
+
+Grey Scale: colors: ["#979798", "#bbbdc0", "#58585b", "#e6e7e8", "#231f20"]
+Green: colors: ["#00a59a", "#9bc863", "#1a6b61", "#e9ea7b", "#1b443d"]
+Blue: colors: ["#04ca6dc", "#65cdf5", "#546ab2", "#b6e2ec", "#20386c"]
+Red: colors: ["#f15e4e", "#faa634", "#a41c3f", "#fde95f", "#5b173b"]
+Varied#1: colors: ["#04858d", "#41beae", "#faa634", "#f15e4e", "#004861"]
+Varied#2: colors: ["#f7961c", "#87acdb", "#ffc60a", "#a2ce5a", "#f15e4e"]
+Varied#3: colors: ["#f15e4e", "#acdee4", "#73c163", "#ffc74a", "#41beae"]
+
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("floe.chartAuthoring.demo");
+
+    floe.chartAuthoring.demo.addExampleData = function (that) {
+        var initialData = that.options.initialData;
+        fluid.each(initialData.inputSelectors, function(currentSelector, idx) {
+            var dataEntry = that.chartAuthoringInterface.dataEntryPanel[currentSelector];
+            var sampleData = initialData.data[idx];
+            dataEntry.locate("label").val(sampleData.label).trigger("change");
+            dataEntry.locate("value").val(sampleData.value).trigger("change");
+        });
+    };
+
+
+    fluid.defaults("floe.chartAuthoring.demo", {
+        gradeNames: ["floe.chartAuthoring"],
+        listeners: {
+            "onToolReady.addExampleInput": {
+                funcName: "floe.chartAuthoring.demo.addExampleData",
+                args: ["{that}"]
+            }
+        },
+        initialData: {
+            inputSelectors: ["dataEntry", "dataEntry-1", "dataEntry-2"],
```

Comment:
Agreed. Best strategy would be something like the following:

- locate all the dynamically generated selector IDs for the inputs
- for each item in initialData.data, initialize the corresponding input

Will work on implementing that.